resources/twitchcommands.py

```diff
@@ -443,8 +445,13 @@ def gamecontrol():
             sendForm("(set! (-> *pc-settings* third-camera-h-inverted?) #t)(set! (-> *pc-settings* third-camera-v-inverted?) #t)(set! (-> *pc-settings* first-camera-v-inverted?) #t)(set! (-> *pc-settings* first-camera-h-inverted?) #f)")
             message = ""
 
+        if PREFIX + "cam" == str(args[0]).lower() and len(args) >= 2 and on_check("cam") and cd_check("cam"):
+            activate("cam")
+            sendForm("(send-event *camera* 'change-state cam-" + str(args[1]) + " 0)(send-event *target* 'no-look-around (seconds " + str(durations[command_names.index("cam")]) + "))")
```

Comment:
hopefully this just does nothing if the user gives a bad cam mode?

Reply:
yeah it just gives a repl error and nothing breaks from what I've seen. It still starts the cooldown but that's something I haven't really fixed for any command yet. same thing with the press command